app/src/main/java/com/ghomebyrw/gworker/activities/LoginActivity.java

```diff
@@ -0,0 +1,101 @@
+package com.ghomebyrw.gworker.activities;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.ghomebyrw.gworker.R;
+import com.ghomebyrw.gworker.clients.JobClient;
+
+import java.net.PasswordAuthentication;
+
+import retrofit.Callback;
+import retrofit.Response;
+import retrofit.Retrofit;
+
+public class LoginActivity extends AppCompatActivity {
+
+    private EditText etEmail;
+    private EditText etPassword;
+    private Button logInButton;
+    private JobClient jobClient;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        getSupportActionBar().hide();
+
+        jobClient = new JobClient();
+        etEmail = (EditText) findViewById(R.id.etEmailAddress);
+        etEmail.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                etEmail.setText("");
+            }
+        });
+        etPassword = (EditText) findViewById(R.id.etPassword);
+        etPassword.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                etPassword.setText("");
+            }
+        });
+        logInButton = (Button) findViewById(R.id.btnLogin);
+        logInButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                String emailAddress = etEmail.getText().toString();
+                String password = etPassword.getText().toString();
+                if (emailAddress.isEmpty() || password.isEmpty()) {
+                    Toast.makeText(LoginActivity.this, getString(R.string.email_password_not_null),
+                            Toast.LENGTH_SHORT).show();
+                } else {
+                    jobClient.logIn(emailAddress, password, new Callback<Boolean>() {
+                        @Override
+                        public void onResponse(Response<Boolean> response, Retrofit retrofit) {
+                            Intent intent = new Intent(LoginActivity.this, AccountActivity.class);
```

Comment:
What's the rationale for going to the Account activity after login? I'd think the user would rather see his or her jobs.